tests/snapshots/snap_test_spacy_affixes.py

```diff
@@ -7,12 +7,51 @@
 
 snapshots = Snapshot()
 
+snapshots['test_accent_exceptions 1'] = [
+    [
+        'Ese',
+        'Ese',
+        False,
+        None,
+        None,
+        None,
+        0
+    ],
+    [
+        'hombre',
+        'hombre',
+        False,
+        None,
+        None,
+        None,
+        0
+    ],
+    [
+        'está',
+        'estar',
+        False,
+        None,
+        None,
+        None,
+        0
+    ],
+    [
+        'demente',
+        'demente',
+        False,
+        None,
+        None,
+        None,
+        0
+    ]
+]
+
 snapshots['test_split_on_all 1'] = [
     [
         'Cuénta',
         'contar',
         'VERB',
-        'VERB__Mood=Imp|Number=Sing|Person=2|VerbForm=Fin',
+        'Mood=Imp|VerbForm=Fin|Person=2|Number=Sing',
```

Comment:
Don't tags output the PoS before the actual tags? Before, `.tag_` used to have both `POS_tag|tag|tag`, is it that know `.tag_` only has `tag|tag|tag`?